README.md

```diff
@@ -87,6 +87,37 @@ print(list(sorted(ng.optimizers.registry.keys())))
 
 The [optimization documentation](docs/optimization.md) contains more information on how to use several workers, take full control of the optimization through the `ask` and `tell` interface and some pieces of advice on how to choose the proper optimizer for your problem.
 
+
+## Example of chaining, or inoculation, or initialization of an evolutionary algorithm
+
+Chaining consists in running several algorithms in turn, information being forwarded from the first to the second and so on.
+More precisely, the budget is distributed over several algorithms, and when an objective function value is computed, all algorithms are informed.
+In optimizerlib.py, you can read the following:
+```
+DEwithLHS = chaining([LHSSearch, DE], [-1])  # Runs LHSSearch with budget num_workers and then DE.
+DEwithLHSdim = chaining([LHSSearch, DE], [-2])  # Runs LHSSearch with budget the dimension and then DE.
+DEwithLHS30 = chaining([LHSSearch, DE], [30])  # Runs LHSSearch with budget the dimension and then DE.
+```
+
+We could also do 
+```
+my_chain = chaining([LHSSearch, DE, CMA], [100, 60, 1000])
```

Comment:
I don't understand this syntax, since then we need to provide a global budget, how is it supposed to be used?